### ISSUE TITLE
[Bug] Sidebar with collapsable first child is auto expanded on the base path

### ISSUE THREAD

**Issue by vordimous** [vuepress-theme-hope/vuepress-theme-hope#4286]
### Checklist

- [X] I am using **LTS** version of Node.js.
- [X] I am using the **latest v2 version** of VuePress and VuePress official plugins.
- [X] I am using the **latest v2 version** of vuepress-theme-hope and all plugins in this repo.
- [X] I followed the docs and I double checked my configuration.
- [X] I have searched [FAQ](https://theme-hope.vuejs.press/faq/) and I did not find the answer.

### Package name

vuepress-theme-hope

### Which operating system are you using?

- [X] macOS
- [ ] Windows
- [ ] Linux

### Describe the bug

When the first child of a sidebar item is expandable it is open by default on the base path: 
<img width="473" alt="image" src="https://github.com/user-attachments/assets/a8aa9fe3-f8f0-4bb3-86af-48a37696bf53">

- base path: <https://docs.aklivity.io/zilla/latest/>
<img width="424" alt="image" src="https://github.com/user-attachments/assets/2ecff85d-b352-42b4-95d4-80ec4d5a077f">

- other path: <https://docs.aklivity.io/zilla/latest/how-tos/quickstart/>
<img width="479" alt="image" src="https://github.com/user-attachments/assets/f3aa636c-fb85-4770-b26a-d2850acc3b9c">

### Additional context

Same behavior on new project generated with `"vuepress-theme-hope": "2.0.0-rc.52",`.

**Comment by Mister-Hope** [comment]
Probably the same problem as https://github.com/vuepress-theme-hope/vuepress-theme-hope/issues/4223.

Sadly I am not finding the actual bug in my code, so may someone could review my code and poitn out the bug.

**Comment by vordimous** [comment]
> Sadly I am not finding the actual bug in my code, so may someone could review my code and poitn out the bug.

IDK if it helps find the bug but I was able to prevent the behavior by adding this workaround:

```scss
.vp-sidebar-header:has([aria-label^="-hidden-"]) {
  display: none !important;
}
```

**Comment by Mister-Hope** [comment]
Thanks for investigating issue, really interested to hear that this is a a11y related bug, I will take a look later.

**Comment by vordimous** [comment]
I think this is not supposed to be stale since it is a bug?

**Comment by Mister-Hope** [comment]
Yes, github stale action has not supported "Type" yet, so since the bug label is converted to `Type: bug`, now it will convert some issue by mistake.

**Comment by Mister-Hope** [comment]
I actually investigate this earlier, but I failed to find any reference about this behavior.
